web/src/hooks/useCheckAvailableFeature.ts

```diff
@@ -0,0 +1,22 @@
+import { useMemo } from 'react';
+import { useUser } from 'web/src/components/app/AppContext';
+import { useFetchPublicFeatures } from 'web/src/hooks/data/barong/useFetchPublicFeatures';
+import { FeatureMap } from 'web/src/types/featuresToggling.types';
+
+export const useCheckAvailableFeature = (name: string) => {
```

Comment:
```suggestion
export const useAvailableFeature = (name: string) => {
```
или
```suggestion
export const useFeature = (name: string) => {
```
Не пойму как лучше. Есть идеи @YuriFA ? 😀

Reply:
`export const useFeatureEnabled = (name: string) => {`

так понятнее кажется

Reply:
Изменил